streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -471,7 +471,7 @@ public void shouldCloseProcessor() {
 
     @Test
     public void shouldThrowForUnknownTopic() {
-        testDriver = new TopologyTestDriver(new Topology(), config);
+        testDriver = new TopologyTestDriver(new Topology());
```

Comment:
the ```config``` is not from empty properties so it may be incorrect to remove ```config```.

Reply:
there are a lot of similar code change.

Reply:
Default config in the test provides [application_id and state_dir_config](https://github.com/apache/kafka/blob/324c0ea0ae2c4ade0f5320ac2e794b020f57ab52/streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java#L145-L146) parameters. Majority of the tests like [shouldThrowForUnknownTopic](https://github.com/apache/kafka/blob/324c0ea0ae2c4ade0f5320ac2e794b020f57ab52/streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java#L473),  [shouldThrowForMissingTime](https://github.com/apache/kafka/pull/9660/files#diff-a10cd21c3ae928cc682ee13fa43fa02168fdebb5a319ccf8ecf5d69daac1de19L487),  [shouldThrowNoSuchElementExceptionForUnusedOutputTopicWithDynamicRoutingdon't](https://github.com/apache/kafka/pull/9660/files#diff-a10cd21c3ae928cc682ee13fa43fa02168fdebb5a319ccf8ecf5d69daac1de19R519),  [shouldCaptureSinkTopicNamesIfWrittenInto](https://github.com/apache/kafka/pull/9660/files#diff-a10cd21c3ae928cc682ee13fa43fa02168fdebb5a319ccf8ecf5d69daac1de19R533),  [shouldProcessRecordForTopic](https://github.com/apache/kafka/pull/9660/files#diff-a10cd21c3ae928cc682ee13fa43fa02168fdebb5a319ccf8ecf5d69daac1de19R592), [shouldSetRecordMetadata](https://github.com/apache/kafka/pull/9660/files#diff-a10cd21c3ae928cc682ee13fa43fa02168fdebb5a319ccf8ecf5d69daac1de19R604) don't require these config parameters.                                                                                                                                                                    But tests using state stores such as [shouldPopulateGlobalStore](https://github.com/apache/kafka/pull/9660/files#diff-a10cd21c3ae928cc682ee13fa43fa02168fdebb5a319ccf8ecf5d69daac1de19R911), [shouldReturnAllStores](https://github.com/apache/kafka/pull/9660/files#diff-a10cd21c3ae928cc682ee13fa43fa02168fdebb5a319ccf8ecf5d69daac1de19R1032) and [shouldThrowIfBuiltInStoreIsAccessedWithUntypedMethod](https://github.com/apache/kafka/pull/9660/files#diff-a10cd21c3ae928cc682ee13fa43fa02168fdebb5a319ccf8ecf5d69daac1de19R1156) may need specific dir config to clean up after the test. Right now I am relying on default state-dir-config value for these few tests which use state stores. I am open to change those specific tests which use state stores.